pkgs/applications/networking/browsers/vivaldi/ffmpeg-codecs.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, fetchpatch
+, dbus_glib, gtk2, gtk3, libexif, libpulseaudio, libXScrnSaver, ninja, nss
+, pciutils, pkgconfig, python2, xdg_utils
+}:
+
+stdenv.mkDerivation rec {
+  name = "${product}-${version}";
+  product = "vivaldi-ffmpeg-codecs";
+  version = "59.0.3071.104";
```

Comment:
It's not possible for the packages to go out of sync now, the vivaldi build will fail if vivaldi-ffmpeg-codecs isn't built from the same Chromium version.

A single project for both would probably mean rebuilding ffmpeg even when Vivaldi didn't update its Chromium version. As the Chromium source is pretty large (~3.5 GB) and compiles rather slowly, I'd like to avoid that.